src/terraform/storage.tf

```diff
@@ -0,0 +1,22 @@
+
+data "openstack_identity_auth_scope_v3" "current" {
+  name = "current"
+}
+
+resource "openstack_objectstorage_container_v1" "joanie_media_storage" {
+  name          = "tf-${terraform.workspace}-joanie-media-storage"
```

Comment:
should we still use `${terraform.workspace}` ? This terraform project is not aim to use workspace anymore.

Reply:
As I tried to explain in the documentation, there are 2 dimensions:
- the openstack project which is a tenant
- the workspace which allows creating several media files in the same openstack project. For example at FUN we will create staging and preprod in the same openstack project.